src/main/java/com/katalon/plugin/jira/composer/toolbar/handler/ImportJiraJQLHandler.java

```diff
@@ -171,6 +177,16 @@ protected IStatus run(IProgressMonitor monitor) {
                 }
             }
 
+            private String truncateName(FolderEntity parentFolder, String name) throws IndexOutOfBoundsException {
+                int taken = parentFolder.getFileLocation().length() + PreferenceConstants.FILE_SEPAPRATOR_LENGTH
+                        + PreferenceConstants.MAX_SUFFIX_LENGTH + PreferenceConstants.FILE_SEPAPRATOR_LENGTH
+                        + PreferenceConstants.GROOVY_SCRIPT_FILE_NAME_LENGTH;
+                int available = PreferenceConstants.MAX_FILE_PATH_LENGTH - taken;
+                name = name.trim();
```

Comment:
Please export to another variable to prevent input change. https://www.baeldung.com/java-functional-programming